src/otx/algo/instance_segmentation/heads/rtmdet_ins_head.py

```diff
@@ -530,14 +530,14 @@ def _mask_predict_by_feat_single(self, mask_feat: Tensor, kernels: Tensor, prior
                 x = torch.nn.functional.relu(x)
         return x.reshape(num_inst, h, w)
 
-    def loss_mask_by_feat(
+    def get_mask_targets(
```

Comment:
Could we rename to `prepare_mask_loss_inputs`?

Reply:
Done https://github.com/openvinotoolkit/training_extensions/pull/3865/commits/0f4409c90de69578b2e4b09e66f2b5b4c072e436